src/App.jsx

```diff
@@ -10,6 +10,7 @@ import LogIn from "./pages/LogIn";
 
 function Navigation() {
```

Comment:
For organization, think about moving the `Navigation` component into its own file.